release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 20. | **Bug Fix** | Managed Identity Token Service (MITS) fails renegotiation with gateway on systems with TLS 1.3 enabled | **Brief Description**: The Managed Identity Token Service (MITS) will lose its connection to the gateway when deployed on Win2022, and won't attempt to renegotiate it.<br>**Impact**: On Windows 2022 or other situations where TLS1.3 is available/the default, MITS will fail to renegotiate its inbound connections, and will fail all inbound requests. SF-hosted user services attempting to acquire tokens representing their managed identities will experience timeouts.<br>**Fix**: Limit MITS to using TLS 1.2 only, until Windows 2022 becomes a supported OS for Service Fabric.<br>**Workaround**: Note Service Fabric does not list Windows 2022 as a supported OS version at this time - primarily because it defaults to TLS1.3, and the SF runtime needs additional fixes to support this scenario. Our recommendation is to stick with the officially supported OS images/versions. In this particular case, restarting MITS on the failing node will work momentarily, which may be sufficient for an application to obtain its tokens. However, the failing behavior will re-occur within the order of minutes. |
+| 21. | **Bug Fix** | Validation fails when RunToCompletion is specified and ServicePackageActivationMode is SharedProcess | **Brief Description**: New services, including both default services for new applications and new services for existing applications, will fail validation when they are set to RunToCompletion and ServicePackageActivationMode is SharedProcess. Such cases have never been supported by Service Fabric, however, we would accept this services and not honor RunToCompletion, so they would restart on exit.<br>**Impact**: Before, we would accept services set to RunToCompletion where ServicePackageActivationMode was SharedProcess, yet we did not honor RunToCompletion. Now validation will fail in that scenario and all services set to RunToCompletion will honor that execution policy.<br>**Fix**: For default services, validation fails during ApplicationManifest validation. For new services on existing applications, the cluster manager (CM) fails the new service validation before creating it.<br>**Documentation Reference**: Read more about it at -<ul><li>[RunToCompletion](https://docs.microsoft.com/en-us/azure/service-fabric/run-to-completion)</li></ul>**Breaking Change**: New applications and application upgrades where there are default services that RunToCompletion and ServicePackageActivationMode is SharedProcess will fail. New services for existing applications will also fail for such cases. |
```

Comment:
```suggestion
| 21. | **Bug Fix** | Validation fails when RunToCompletion is specified and ServicePackageActivationMode is SharedProcess | **Brief Description**: New services, including both default services for new applications and new services for existing applications, will fail validation when they are set to RunToCompletion and ServicePackageActivationMode is SharedProcess. Such cases have never been supported by Service Fabric; however, we would accept these services and not honor RunToCompletion, so they would restart on exit.<br>**Impact**: Before, we would accept services set to RunToCompletion where ServicePackageActivationMode was SharedProcess, yet we did not honor RunToCompletion. Now validation will fail in that scenario and all services set to RunToCompletion will honor that execution policy.<br>**Fix**: For default services, validation fails during ApplicationManifest validation. For new services on existing applications, the cluster manager (CM) fails the new service validation before creating it.<br>**Documentation Reference**: Read more about it at -<ul><li>[RunToCompletion](https://docs.microsoft.com/en-us/azure/service-fabric/run-to-completion)</li></ul>**Breaking Change**: New applications and application upgrades where there are default services that RunToCompletion and ServicePackageActivationMode is SharedProcess will fail. New services for existing applications will also fail for such cases. |
```

Reply:
^"this services"-->"these services"; replaced comma with semicolon before "however"